lib/media/segment_index.js

```diff
@@ -698,25 +698,29 @@ shaka.media.MetaSegmentIndex = class extends shaka.media.SegmentIndex {
 
     /** @private {!Array.<!shaka.media.SegmentIndex>} */
     this.indexes_ = [];
+
+    /**
+     * Represents the number of evicted segments from segment indexes that have
+     * been evicted out of the indexes array.
+     * @private {number}
+     */
+    this.numEvicted_ = 0
```

Comment:
This exists already in parent class and it's protected, you don't have to reassign that.